tests/safecrt/common/test.cpp

```diff
@@ -1,14 +1,19 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#if defined(OE_BUILD_ENCLAVE)
+#include <openenclave/enclave.h>
```

Comment:
Why do we need this include only on enclave side and not the equivalent for the host side in #else part?

Reply:
`OE_TEST` calls `oe_abort()` on the enclave side which requires `openenclave/enclave.h` to compile. On the other hand, on the host side, it calls the C `abort()`.

Reply:
Thanks for the clarification. That would mean that internal/tests.h is a faulty header that does not include what it depends on.
It would be better to fix internal/tests.h. (It probably got to this state due to all the API refactoring).

Reply:
Yeah we can do that in another PR